src/pheval_exomiser/config_parser.py

```diff
@@ -5,113 +5,6 @@
 from serde import serde
 from serde.yaml import from_yaml
 
-# @serde
-# @dataclass
-# class ExomiserPrepareScramble:
-#     scramble_semsim: float
-#     scramble_phenopacket: float
-#
-#
-# @serde
-# @dataclass
-# class ExomiserPrepareUpdatePhenopacket:
-#     update: bool
-#     gene_identifer_to_update: str
-#
-#
```

Comment:
Where did these all go? Are they part of the generic pheval setup now?

Reply:
This was all part of the first config schema I had thought of which included the "prepare" steps of updating Phenopacket gene identifiers, scrambling phenopackets and spiking the VCFs. I removed these from the config as this is part of the MakeFile now. The functionality for reading the exomiser runner config is located in the `config_parser.py` in this repo as the config for each plugin is likely to be different

Reply:
I though so - was just double-checking!